Clean up auth.json documentation (#1964)
containers/image: docs/containers-auth.json.5.md

# DESCRIPTION

A file in JSON format controlling authentication against container image registries.
The primary (read/write) file is stored at `${XDG_RUNTIME_DIR}/containers/auth.json` on Linux;
on Windows and macOS, at `$HOME/.config/containers/auth.json`.

When searching for the credential for a registry, the following files will be read in sequence until the valid credential is found:
first reading the primary (read/write) file, or the explicit override using an option of the calling application.
If credentials are not present there,

> search continues in `${XDG_CONFIG_HOME}/containers/auth.json` (usually `~/.config/containers/auth.json`), `$HOME/.docker/config.json`, `$HOME/.dockercfg`.

Reviewer: ```suggestion
the search continues in `${XDG_CONFIG_HOME}/containers/auth.json` (usually `~/.config/containers/auth.json`), `$HOME/.docker/config.json`, `$HOME/.dockercfg`.
```

> Except the primary (read/write) file, other files are read-only, unless the user, using an option of the calling application, explicitly points at it as an override.

Reviewer: ```suggestion
Except for the primary (read/write) file, other files are read-only unless the user, using an option of the calling application, explicitly points at it as an override.
```

## FORMAT

The auth.json file stores, or references, credentials that allow the user to authenticate
to container image registries.
It is primarily managed by a `login` command from a container tool such as `podman login`,

> `buildah login` or `skopeo login`.

Reviewer: ``` suggestion
`buildah login`, or `skopeo login`.
```

> Each entry contains a single hostname (e.g. `docker.io`) or a namespace (e.g. `quay.io/user/image`) as a key,

Reviewer: ```suggestion
Each entry contains a single hostname (e.g., `docker.io`) or a namespace (e.g., `quay.io/user/image`) as a key,
```

tool such as `podman logout` or `buildah logout`.

> In addition, credential helpers can be configured for specific registries and the credentials-helper

Reviewer: ```suggestion
In addition, credential helpers can be configured for specific registries, and the credentials-helper
```

> software can be used to manage the credentials in a more secure way than stroing credentials only the base64-encoded in `auth.json`.

Reviewer: I think this is what you're going for here:
```suggestion
software can be used to manage the credentials more securely than storing only base64-encoded credentials in `auth.json`.
```

Author: Here, the point is “only (storing) base64-encoded”, not “only credentials”.

Would “… can be used to manage the credentials more securely than only  storing only them base64-encoded in `auth.json`.” work here?
